src/internal/pfsdb/repos.go

```diff
@@ -206,23 +180,6 @@ func listRepoPage(ctx context.Context, tx *pachsql.Tx, limit, offset int, filter
 	return page, nil
 }
 
-// CreateRepo creates an entry in the pfs.repos table.
```

Comment:
Do we have any use cases where we want to strictly create a new repo and want the code to throw an error if the repo already exists?